zapisy/templates/base.html

```diff
@@ -211,7 +213,7 @@
     {% block rendered_bundles %}
     {% endblock %}
     <footer class="d-print-none">
-        <div class="container-fluid">
+        <div class="container-fluid bg-light">
```

Comment:
Też chyba z rozpędu.
```suggestion
        <div class="container-fluid">
```

Reply:
Z jednej strony rzeczywiście klasa ta jest zbędna, z drugiej w pliku`/zapisy/apps/common/assets/main/index.scss`
ustawiamy `background-color`  `container-fluid` na kolor określany przez klasę `bg-light`.

Czy nie sensowniej jest usunąć tą definicję z `index.scss` i zostawić z tym miejscu wspominaną klasę?

`index.scss`:
```
footer {
  ...
  .container-fluid {
    background-color: var(--bs-light);
    ....
```

Reply:
W praktyce to nie ma znaczenia, bo to i tak jedyne wystąpienie `.container-fluid` wewnątrz `footer`, prawda? Z jednej strony zepchnięcie deklaracji koloru do arkusza CSS czyni kod HTML bardziej "semantycznym", z drugiej domyślam się (na podstawie nawet samego istnienia klas takich jak `bg-light`), że bardziej "bootstrapowo" byłoby zrobić tak, jak Pan proponuje, tak?

Reply:
Gotowe